src/main/java/cms/gov/madie/measure/resources/AdminController.java

```diff
@@ -472,7 +472,7 @@ private boolean isDirectAncestor(Measure sourceMeasure, Measure targetMeasure) {
     if (!StringUtils.equals(
         sourceMeasure.getMeasureSet().getMeasureSetId(),
         targetMeasure.getMeasureSet().getMeasureSetId())) {
-      throw new InvalidRequestException("Source measure from different Measure family/set.");
+      return false;
```

Comment:
This can be simplified to a single statement now that we're not throwing the exception.
```suggestion
return StringUtils.equals(
sourceMeasure.getMeasureSet().getMeasureSetId(),
targetMeasure.getMeasureSet().getMeasureSetId()));
```

Reply:
@jkotanchik-SB Good suggestion, I combined the two separate comparison into one. Thanks Joe!